scripts/push-theme-readme.sh

```diff
@@ -9,6 +9,6 @@ git config --global user.name "GitHub Readme Stats Bot"
 git branch -d $BRANCH_NAME || true
 git checkout -b $BRANCH_NAME
 git add --all
-git commit --message "docs(theme): Auto update theme readme" || exit 0
+git commit --no-verify --message "docs(theme): Auto update theme readme"
 git remote add origin-$BRANCH_NAME https://${PERSONAL_TOKEN}@github.com/${GH_REPO}.git
```

Comment:
As far as I can tell, this is unrelated to the rate limiting feature, so it would be best if you could keep these to separate PRs. I also understand that it might be cumbersome to open PRs for small changes, though, so it's fine either way :)

Reply:
Same already on the master branch.